x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/CreateDocsIndexPrivilegeTests.java

```diff
@@ -22,14 +22,14 @@
             "  indices:\n" +
             "    - names: '*'\n" +
             "      privileges: [ all ]\n" +
-        "create_doc_role:\n" +
+            "create_doc_role:\n" +
```

Comment:
Nit: The indentation is now inconsistent with `all_indices_role`. I do think the change here looks better. But I'd suggest we make the same change to `all_indices_role` as well.